src/__tests__/patients/allergies/ViewAllergy.test.tsx

```diff
@@ -19,28 +19,23 @@ describe('View Care Plan', () => {
     jest.spyOn(PatientRepository, 'find').mockResolvedValue(patient)
     const history = createMemoryHistory()
     history.push(`/patients/${patient.id}/allergies/${patient.allergies![0].id}`)
-    let wrapper: any
 
     await act(async () => {
-      wrapper = await mount(
+      await render(
```

Comment:
why await (async) render is needed in this case?

Reply:
I don't think this is needed... You can try pulling the render call outside of the act, remove the act, and see if test still passes?

Reply:
Yeah this is likely someone trying to get rid of act warnings in the console lol

Reply:
Nope! test is failing..

TestingLibraryElementError: Unable to find an element with the display value: /some name/.

The `input` node is not getting rendered.

Reply:
Hmm... I would then try leaving out the act still, but adding a waitFor(() => expect(...)) in the actual test body. @JacobMGEvans ?

Reply:
Tried removing act & await with render and added waitFor surrounding assertions. Test seems to passed.

However two warning
```
(node:13110) UnhandledPromiseRejectionWarning: TypeError: MutationObserver is not a constructor
(node:13110) UnhandledPromiseRejectionWarning: Unhandled promise rejection
```

Reply:
We can come up behind later and deal with cleaning up lol